index.yaml

```diff
@@ -407,12 +300,12 @@ entries:
     name: comet-api
     type: application
     urls:
-    - https://github.com/vivid-planet/comet-charts/releases/download/comet-api-10.5.1/comet-api-10.5.1.tgz
-    version: 10.5.1
```

Comment:
looks good to me, i had the problem in a project that renovate tried to update to the not existing 10.5.1 Version.